mala/network/network.py

```diff
@@ -57,32 +57,27 @@ def __new__(cls, params: Parameters):
         """
         model = None
```

Comment:
This can be skipped since no test `if model is None` is done in the conditional below.

Reply:
Good point, I deleted this.